server.py

```diff
@@ -67,6 +68,12 @@ def Main():
 					songs.append(getFilename)
 				elif ext=="exe":
 					installer.append(getFileName)
+				elif ext == "jpeg":
+					image = ImageGrab.grab()
+					image.save(r'C:\Users\shubham\Desktop\Python\ScreenShots\screen.jpg')        #when image is at local disk
```

Comment:
First of all you can not decide always that the client and server is SHUBHAM always.You have to use some thing global that is valid for all pair of client and user.